core/src/main/java/com/devonfw/tools/solicitor/config/ConfigFactory.java

```diff
@@ -126,8 +126,8 @@ public ModelRoot createConfig(String url) {
         rs.setSource(rc.getSource());
         rs.setUsagePattern(rc.getUsagePattern());
         if (rc.getRepoType() != null) {
-          this.deprecationChecker.check(true,
-              "The parameter 'repoType' in the reader configuration is deprecated, see https://github.com/devonfw/solicitor/issues/190");
+          this.deprecationChecker.check(false,
+              "The parameter 'repoType' in the reader configuration is deprecated, see https://github.com/devonfw/solicitor/issues/263");
```

Comment:
We should list both tickets as the first gives some background and the second indicates that it is now stage 2.